src/main/comments.js

```diff
@@ -214,6 +214,17 @@ function attach(comments, ast, text, options) {
 
     const isLastComment = comments.length - 1 === i;
 
+    const isEnclosedByTernary =
```

Comment:
This is not the right file for JS-specific things. This should be handled in `language-js/comments.js`.